internal/network/command.go

```diff
@@ -130,6 +132,7 @@ func printHumanTable(cmd *cobra.Command, network networkingv1.NetworkingV1Networ
 		Id:                       network.GetId(),
 		EnvironmentId:            network.Spec.Environment.GetId(),
 		Name:                     network.Spec.GetDisplayName(),
+		Gateway:                  network.Spec.GetGateway().Id,
```

Comment:
```suggestion
		Gateway:                  network.Spec.Gateway.GetId(),
```

Reply:
Gateway is of type [NullableTypedEnvScopedObjectReference](https://github.com/confluentinc/ccloud-sdk-go-v2/blob/cee82126eba85378bb60b857133947e4844d5c0a/networking/v1/model_typed_env_scoped_object_reference.go#L308) where we need to go a Get first, the alternative is `network.Spec.Gateway.Get().GetId()`